Adding Option and Environment variable for AzureAuth Cache file (#70)
AzureAD/microsoft-authentication-cli: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased]
### Added

> - Environment variable `AZUREAUTH_CACHE_FILE` and option `--cache` to support custom cache file name.

Reviewer: ```suggestion
- Environment variable `AZUREAUTH_CACHE_FILE` and option `--cache` to support a custom cache location on Windows.
```
